Sources/FluentUI_common/Core/Extensions/UIColor+Extensions.swift

```diff
@@ -3,8 +3,9 @@
 //  Licensed under the MIT License.
 //
 
-import UIKit
+#if canImport(UIKit)
 import SwiftUI
```

Comment:
does anything in this file need swiftui?

Reply:
Ah I guess not anymore! Will remove.